backend/tracim_backend/models/context_models.py

```diff
@@ -126,6 +126,21 @@ def __init__(self, loggedin_user_password: str, email: str) -> None:
         self.email = email
 
 
+class SimpleFile(object):
+    """
+    Simple file object
```

Comment:
useless docstring. Docstring must help understand what this class is usefull for (is it just a container?)... or do not add docstring